### PR TITLE
[zh] update link & sync

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/33743/files
/assign @tengqm

### CODE REVIEW

File: content/zh/blog/_posts/2022-05-16-volume-populators-beta.md [hunk 1]
```diff
@@ -19,16 +19,16 @@ Ben Swartzlander (NetApp)
...
 as the data source of a PVC.
 -->
 卷填充器功能现在已经经历两个发行版本并进入 Beta 阶段！
-在 Kubernetes v1.24 中 `AnyVolumeDataSouce` 特性门控默认被启用。
+在 Kubernetes v1.24 中 `AnyVolumeDataSource` 特性门控默认被启用。
 这意味着用户可以指定任何自定义资源作为 PVC 的数据源。
 
 <!--
-An [earlier blog article](/blog/2021/08/30-volume-populators-redesigned/) detailed how the
+An [earlier blog article](/blog/2021/08/30/volume-populators-redesigned/) detailed how the
```
Comment (Sea-n) [inline content/zh/blog/_posts/2022-05-16-volume-populators-beta.md:31]: Please fix line 36 for Mandarin content too.

Comment (Arhell) [reply content/zh/blog/_posts/2022-05-16-volume-populators-beta.md:31]: Done

**Review by Sea-n — COMMENTED** [review]
Thanks, looks good to me!
/lgtm

**Pull request merged**